scripts/editor/register-blocks.js

```diff
@@ -472,18 +472,29 @@ export const getExample = (
 /**
  * Map and prepare all options from block manifest.json file for usage in registerBlockVariation method.
  *
- * @param {object} globalManifest - Global manifest.
- * @param {object} blockManifest  - Block manifest.
+ * @param {object} globalManifest     - Global manifest.
+ * @param {object} blockManifest      - Block manifest.
+ * @param {Array} allBlocksManifests  - All Blocks manifests.
  *
  * @returns {object}
  */
 const registerVariation = (
 	globalManifest = {},
 	blockManifest = {},
+	allBlocksManifests,
```

Comment:
please provide the default value so it wound break implementations on the old projects